src/firecracker/src/api_server/mod.rs

```diff
@@ -1,14 +1,19 @@
-// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
For these we generally keep the year the file was created, even when we move them around